exchange/interface.go

```diff
@@ -22,5 +22,7 @@ type Interface interface {
 	// available on the network?
 	HasBlock(context.Context, *blocks.Block) error
 
+	GetWantlist() []u.Key
```

Comment:
let's not add this here. not all exchanges will have a wantlist (think of HTTP as a possible exchange)